conseil-lorre/src/main/scala/tech/cryptonomic/conseil/indexer/tezos/processing/BlocksProcessing.scala

```diff
@@ -0,0 +1,80 @@
+package tech.cryptonomic.conseil.indexer.tezos.processing
+
+import com.typesafe.scalalogging.LazyLogging
+import scala.util.{Failure, Success, Try}
+import scala.concurrent.{ExecutionContext, Future}
+import slick.jdbc.PostgresProfile.api._
+
+import tech.cryptonomic.conseil.indexer.tezos.{
+  TezosNamesOperations,
+  TezosNodeOperator,
+  TezosGovernanceOperations,
+  TezosDatabaseOperations => TezosDb
+}
+import tech.cryptonomic.conseil.common.tezos.TezosTypes.{Block, Voting}
+import tech.cryptonomic.conseil.common.tezos.TezosTypes.Syntax._
+import tech.cryptonomic.conseil.indexer.tezos.michelson.contracts.TokenContracts
+import tech.cryptonomic.conseil.indexer.tezos.michelson.contracts.TNSContract
+
+class BlocksProcessing(
+    nodeOperator: TezosNodeOperator,
+    db: Database,
+    tnsOperations: TezosNamesOperations,
+    accountsProcessing: AccountsProcessing,
+    bakersProcessing: BakersProcessing
+)(implicit tokens: TokenContracts, tns: TNSContract)
+    extends LazyLogging {
+
+  /* will store a single page of block results */
+  private[tezos] def processBlocksPage(results: nodeOperator.BlockFetchingResults)(
+      implicit ec: ExecutionContext
+  ): Future[Int] = {
+    def logBlockOutcome[A]: PartialFunction[Try[Option[A]], Unit] = {
+      case Success(accountsCount) =>
+        logger.info(
+          "Wrote {} blocks to the database, checkpoint stored for{} account updates",
```

Comment:
Small thing, missing whitespace between `for` and `{}`

Reply:
Assume results.size = 10
1. accountsCount = None =>
  Wrote 10 blocks to the database, checkpoint stored for account updates
2. accountsCount = Some(2)
  Wrote 10 blocks to the database, checkpoint stored for 2 account updates

Reply:
Tricky, now I see :)